content/posts/2022/retention-rules.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Automatically delete workspace files with retention rules"
+date: 2022-11-29T10:30:00+01:00
+---
+
+It can be useful to be able to keep some files in your workflow's workspace only
+for a limited period of time, for example if the workflow generates huge
+temporary files that consume your
+[disk quota limit](https://docs.reana.io/advanced-usage/user-quotas). For this
+reason, it is now possible to define workspace file retention rules to configure
+the automatic deletion of files inside workspaces.
+
+<!--more-->
+
+You can define retention rules in the `reana.yaml` specification file of your
+workflow. For example, this is how you would specify that you would like to
+delete `tmp/*.root` files after seven days and `data/*.csv` files after thirty:
+
+```yaml
+workspace:
+  retention_days:
+    tmp/*.root: 7
+    data/*.csv: 30
+```
+
+Please note that files and directories specified as inputs or outputs of your
+workflow will not be deleted, even if they match one of the retention rules.
+This is to make sure that your workflows can be reproduced even after applying
+the retention rules.
+
+Once you run your workflow, you will be able to check the status of the rules
```

Comment:
For any of your workflow runs, you will be able to verify the active file retention information status though the web interface, where you will also find the information about any scheduled file deletion times: